browser-sign-in/app/src/main/java/com/okta/android/samples/browser_sign_in/data/LoginViewModel.kt

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright 2022-Present Okta, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.okta.android.samples.browser_sign_in.data
```

Comment:
I would advocate for organizing by feature, rather than by layer. In a sample app, it's typically very small, and won't really matter. But I'll still advocate for it! 

In practice this would mean rather than having packages for views and data, we'd have packages for login/dashboard instead.